_posts/2020-06-10-cockpit-221.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Cockpit 221
+author: mvo
+date: 2020-06-10
+tags: cockpit linux
+slug: cockpit-221
+category: release
+summary: Cockpit with improved ...
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 221.
+
+### Support for Cross-Origin-Resource-Policy
+
+Cockpit's web server now sets the Cross-Origin-Resource-Policy header
+to same-origin. Web browsers use this header to prevent other web
+sites from loading individual HTML, JavaScript, or image resources
+from cockpit via `<img>`, `<script>`, or similar tags. This is mostly
+a precaution -- currently there are no known vulnerabilities with
+including resources that way, but as this is not a legitimate use
+case, there is no reason to allow it.
+
+### Accounts: Some buttons are hidden when access is limited
+
+Instead of disabling the appropriate buttons them when the logged in
+user doesn't have administrative access, they will now simply
+disappear.  This is a general theme that we want to roll out all over
+Cockpit.  The Storage and Network pages have already been changed like
+this in earlier releases.
+
+### Developers: Importing "base1/patternfly.css" is deprecated
+
+This pre-compiled stylesheet will be dropped in the future in favor of
```

Comment:
I updated the description in https://github.com/cockpit-project/cockpit/pull/14105 a bit.